model/tensorflow_hub/dffml_model_tensorflow_hub/text_classifier.py

```diff
@@ -0,0 +1,364 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2019 Intel Corporation
+"""
+Description of what this model does
+"""
+import abc
+import inspect
+from typing import AsyncIterator, Tuple, Any, List, Optional, NamedTuple, Type
+import os
+import hashlib
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
```

Comment:
NO WAY!!!! Thank god you found this.

Reference for later: https://github.com/tensorflow/tensorflow/issues/31870